pkg/core/ts/utils/paths.ts

```diff
@@ -41,7 +41,7 @@ export const PathMapper = class {
       let base = document.URL;
       // if document URL has a filename, remove it
       if (base[base.length-1] !== '/') {
-        base = base.split('/').slice(0, -1).join('/');
+        base = `${base.split('/').slice(0, -1).join('/')}/`;
```

Comment:
Hrm, this is low-level, but ok!

Reply:
otherwise then the `localRelative` starts with `../`, we loose an extra folder (because `base` wasn't ending with /)